### ISSUE TITLE
Smooth lines comprising many nodes are not rendered properly as `svg` 

### ISSUE THREAD

**Issue by laggvar** [jheinen/GR.jl#546]
Few points look as expected
```
Plots.plot(sin.(0:0.05:2π))
```
![image](https://github.com/jheinen/GR.jl/assets/56767565/e9a7968c-458e-46c1-868f-db2c9eb10146)
When using many points, the line is not rendered very well
```
Plots.plot(sin.(0:0.00005:2π))
```
![image](https://github.com/jheinen/GR.jl/assets/56767565/565d582a-6094-4c19-913b-65b0a257333d)

Adding some negligible noise brings the line back
```
Plots.plot(sin.(0:0.00005:2π) .+ 0.0001randn(length(0:0.00005:2π)))
```
![image](https://github.com/jheinen/GR.jl/assets/56767565/48f4023a-0a60-4fed-89a6-13a776dbc407)

Figures generated with Plots, rendered in VS Code plot pane, but I checked that the problem persists when using GR.jl directly. 

I would suspect that the problem is related to the number of nodes per square, since the steep parts have another intensity than the inflection points.

**Comment by jheinen** [comment]
The resulting SVG files for those plots look fine, both in Plots and plain GR.
```julia
using GR
plot(sin.(0:0.00005:2π))
savefig("sin.svg")
```
Screen shot for `sin.svg` when rendered in a browser (here Safari):
<img width="474" alt="Screenshot 2024-05-20 at 04 13 28" src="https://github.com/jheinen/GR.jl/assets/5718244/b883d90a-5077-4d59-889d-2c5238f01bfc">
If you get a different result (outside VScode) please let. me know.

**Comment by laggvar** [comment]
I had checked that I saw the same issue in saved `svg`'s too, but it seems to be not identical to VS Code. 

Exported `svg`, rendered by MS Edge. 
![image](https://github.com/jheinen/GR.jl/assets/56767565/4b4a6c92-5668-44f1-8e7d-107fc21375e9)

It looks good when deactivating graphics acceleration though
![image](https://github.com/jheinen/GR.jl/assets/56767565/860adb52-5e94-4809-9af5-0f3325c56d17)

Similarly, deactivating gpu acceleration in VS Code solves the problem. 

In firefox, it looks correct regardsless of gpu acceleration on/off.
